core/genesis.go

```diff
@@ -300,7 +300,9 @@ func (g *Genesis) toBlock(db ethdb.Database, triedb *triedb.Database) *types.Blo
 		}
 	}
 
-	statedb.Commit(0, false)
+	if _, err := statedb.Commit(0, false); err != nil {
+		panic(fmt.Sprintf("unable to commit genesis block: %v", err))
```

Comment:
nit just to differentiate from the same panic message below

```suggestion
		panic(fmt.Sprintf("unable to commit genesis block to statedb: %v", err))
```